routes/search.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const router  = express.Router();
+const request = require('request');
+const Song = require('../models/song.js')
+
+router.post('/', async function(req, res){
```

Comment:
@MacInnes - Yes, this async await looks good🎉 I would just recommend using more semantic naming - replacing that `promise` variable with something like `songRequest` maybe? Think of someone who doesn't know the codebase at all coming into it - how can you make it as easy as possible to navigate?